src/frontend/dpia-webapp/index.html

```diff
@@ -8,6 +8,6 @@
   </head>
   <body>
     <div id="app"></div>
-    <script type="module" src="/src/main.js"></script>
+    <script type="module" src="/src/main.ts"></script>
```

Comment:
Why is this main.ts? Is not typescript compiled to .js? Has this been tested?

Reply:
test frontend before committing. the main file index.html has a main.ts (typescript). I am not sure if the vue compiler will reformat .ts to .js on index.html. Browsers do not understand .ts files.

Reply:
This has to do with project init and me choosing whether or not vue has typescript support. Try doing a project init on your local outside of the repo and you can see. This diff is generated by me originally choosing no typescript support over typescript support.